### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by ascott1**
This fleshes out the README with full installation instructions and a link to the documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,32 +3,95 @@
...
+```
+
+### Database setup (for development)
+
+Login to MySQL and create a database named `eregs`  with an `eregs` superuser.
+
+```
+mysql -u root -p
+mysql> CREATE DATABASE IF NOT EXISTS eregs;
+mysql> CREATE DATABASE IF NOT EXISTS test_eregs;
```
Comment (chosak) [inline README.md]: This line should be removed as this database is created by Django automatically when unit tests are run.

File: README.md [hunk 2]
```diff
@@ -3,32 +3,95 @@
...
+mysql> CREATE USER 'eregs'@'localhost' IDENTIFIED BY 'eregs';
+mysql> GRANT ALL PRIVILEGES ON eregs.* TO 'eregs'@'localhost';
+mysql> GRANT ALL PRIVILEGES ON test_eregs.* TO 'eregs'@'localhost';
+mysql> FLUSH PRIVILEGES;
+```
+
+
+### Import a regulation
+
+The application requires [regulation xml](https://github.com/cfpb/regulations-xml) data. To simplify development, sample data has been provided. To load the sample data into the database run:
```
Comment (chosak) [inline README.md]: Use term "RegML" instead/in addition?

File: README.md [hunk 3]
```diff
@@ -3,32 +3,95 @@
 [![Build Status](https://travis-ci.org/cfpb/eregs-2.0.svg?branch=master)](https://travis-ci.org/cfpb/eregs-2.0)
 [![Coverage Status](https://coveralls.io/repos/github/cfpb/eregs-2.0/badge.svg?branch=master)](https://coveralls.io/github/cfpb/eregs-2.0?branch=master)
 
-This is the new version of eRegs.
+Work in progress. This is an updated back-end for consumerfinance.gov/eregulations
```
Comment (chosak) [inline README.md]: Make this a full link, e.g. https://www.consumerfinance.gov/eregulations/?

File: README.md [hunk 4]
```diff
@@ -3,32 +3,95 @@
...
+workon eregs
+pip install -r requirements.txt
+```
+
+### Database setup (for development)
+
+Login to MySQL and create a database named `eregs`  with an `eregs` superuser.
+
+```
+mysql -u root -p
```
Comment (chosak) [inline README.md]: Maybe remove the `-u root -p` as that assumes a certain set of login credentials for MySQL (root user has no password) which might not always be true.

**Review by chosak — APPROVED** [review]
Great, thanks for updating! Added a few minor comments.

File: README.md (hunk 4 above)
Comment (grapesmoker) [reply README.md]: You do need to log in to MySQL somehow, though. I think it's a safe assumption that anyone doing local development and installing MySQL via homebrew has root access to it; if they have a non-blank password, they'll be prompted so I don't think this is a problem.

File: README.md (hunk 2 above)
Comment (grapesmoker) [reply README.md]: Maybe say, "RegML-compliant xml."

**Review by grapesmoker — CHANGES_REQUESTED** [review]
We should ditch the creation of the test database as per Andy's comment and probably say something like "RegML-compliant xml" where noted below. Otherwise I think it's great.

**Review by grapesmoker — APPROVED** [review]

**Comment by ascott1** [comment]
Thanks for the review @chosak and @grapesmoker! I just pushed changes that should address all of your recommendations.

**Pull request merged**